backend/src/db-utils.js

```diff
@@ -562,20 +562,16 @@ const getLastYesterdayBlock = async (timestamp) => {
 
 // pass 'days' to set period of calculation
 const calculateFeesByDay = async (days = 1) => {
-  return await queryRows(
+  return await querySingleRow(
```

Comment:
We should validate the input arguments to prevent misuse:

```suggestion
  if (!(days >= 1 && days <= 7)) {
    throw Exception("calculateFeesByDay can only handle `days` values in range 1..7")
  }
  return await querySingleRow(
```